guide/06-imagery-and-raster-analysis/lidar_create_hillshade_using_gdal_part2b.ipynb

```diff
@@ -0,0 +1,642 @@
+{
```

Comment:
<p style="user-select: auto;">Is this the data used in earlier Part 4 as well? If yes, can we move this to an earlier part of the series?</p><p style="user-select: auto;"><br></p><p style="user-select: auto;">I also think we will check this Part 5 into the Geosaurus repo instead of this repo for now.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/610/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>